lib/consignment.js

```diff
@@ -506,14 +505,12 @@ var createStockOrder = function(args, connectionInfo, retryCounter) {
   var path = '/api/consignment';
   var vendUrl = 'https://' + connectionInfo.domainPrefix + '.vendhq.com' + path;
   var authString = 'Bearer ' + connectionInfo.accessToken;
-  log.silly('Authorization: ' + authString); // TODO: sensitive data ... do not log?
 
   var body = {
     'type': 'SUPPLIER',
     'status': 'OPEN',
     'name': args.name.value,
-    'date': moment().format('YYYY-MM-DD HH:mm:ss'), //'2010-01-01 14:01:01',
-    'due_at': args.dueAt.value,
+    'due_at': args.dueAt.value, // caller must format it as `YYYY-MM-DD HH:mm:ss', example: '2010-01-01 14:01:01'
```

Comment:
I couldn't find this attribute in the tests. But the test still doesn't fail for some reason.

Reply:
there is no reason for the test to fail if the user doesn't specify this